src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -33,7 +33,7 @@ Whether you're on the [original user model](https://docs.newrelic.com/docs/accou
 
 If your organization was created before July 30, 2020 and you haven't transitioned to our New Relic One pricing model, you're on the [original pricing plan](https://docs.newrelic.com/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing/). Capabilities for our [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/) differ from those of our [new user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/):
 
-| | Basic user | Full user without the Nerdpack Manager add-on role | Full user with the Nerdpack Manager add-on role | Owner/Admin user |
+| | Basic user | Full platform user without the Nerdpack Manager add-on role | Full platform user with the Nerdpack Manager add-on role | Owner/Admin user |
```

Comment:
```suggestion
| | Basic user | Full platform user/Core user without the Nerdpack Manager add-on role | Full platform user/Core user with the Nerdpack Manager add-on role | Owner/Admin user |
```

Could you also add 'core user' alongside 'full platform user' throughout this guide? they'll have same permissions according to that ticket.